tests/Unit/Evolution/Systems/GrMhd/GhValenciaDivClean/Subcell/Test_InitialDataTci.cpp

```diff
@@ -0,0 +1,144 @@
+  }
+
+  {
+    INFO("Two mesh RDMP fails");
+    const InactiveConsVars subcell_vars{subcell_mesh.number_of_grid_points(),
+                                        2.0};
+    CHECK(grmhd::GhValenciaDivClean::subcell::DgInitialDataTci::apply(
+        dg_vars, subcell_vars, delta0, epsilon, exponent, dg_mesh,
+        tci_options));
+  }
+
+  {
+    INFO("Persson TCI TildeD fails");
+    const InactiveConsVars subcell_vars{subcell_mesh.number_of_grid_points(),
+                                        1.0};
+    get(get<grmhd::ValenciaDivClean::Tags::TildeD>(
+        dg_vars))[dg_mesh.number_of_grid_points() / 2] += 2.0e10;
+    CHECK(grmhd::GhValenciaDivClean::subcell::DgInitialDataTci::apply(
+        dg_vars, subcell_vars, 1.0e100, epsilon, exponent, dg_mesh,
+        tci_options));
+    get(get<grmhd::ValenciaDivClean::Tags::TildeD>(
+        dg_vars))[dg_mesh.number_of_grid_points() / 2] = 1.0;
+  }
+
+  {
+    INFO("Persson TCI TildeB fails");
+    const InactiveConsVars subcell_vars{subcell_mesh.number_of_grid_points(),
+                                        1.0};
+    for (size_t i = 0; i < 3; ++i) {
+      get<grmhd::ValenciaDivClean::Tags::TildeB<>>(dg_vars).get(
+          i)[dg_mesh.number_of_grid_points() / 2] += 2.0e10;
+    }
+    CHECK(grmhd::GhValenciaDivClean::subcell::DgInitialDataTci::apply(
+        dg_vars, subcell_vars, 1.0e100, epsilon, exponent, dg_mesh,
+        tci_options));
+    for (size_t i = 0; i < 3; ++i) {
+      get<grmhd::ValenciaDivClean::Tags::TildeB<>>(dg_vars).get(
+          i)[dg_mesh.number_of_grid_points() / 2] = 1.0;
+    }
+  }
+
+  {
+    INFO("Persson TCI TildeTau fails");
+    const InactiveConsVars subcell_vars{subcell_mesh.number_of_grid_points(),
+                                        1.0};
+    get(get<grmhd::ValenciaDivClean::Tags::TildeTau>(
+        dg_vars))[dg_mesh.number_of_grid_points() / 2] += 2.0e10;
+    CHECK(grmhd::GhValenciaDivClean::subcell::DgInitialDataTci::apply(
+        dg_vars, subcell_vars, 1.0e100, epsilon, exponent, dg_mesh,
+        tci_options));
+    get(get<grmhd::ValenciaDivClean::Tags::TildeTau>(
+        dg_vars))[dg_mesh.number_of_grid_points() / 2] = 1.0;
+  }
+
+  {
+    INFO("Negative TildeD");
+    InactiveConsVars subcell_vars{subcell_mesh.number_of_grid_points(), 1.0};
+
+    get(get<grmhd::ValenciaDivClean::Tags::TildeD>(
+        dg_vars))[dg_mesh.number_of_grid_points() / 2] = -1.0e-20;
+    CHECK(grmhd::GhValenciaDivClean::subcell::DgInitialDataTci::apply(
+        dg_vars, subcell_vars, 1.0e100, epsilon, 1.0, dg_mesh,
+        tci_options));
+    get(get<grmhd::ValenciaDivClean::Tags::TildeD>(
+        dg_vars))[dg_mesh.number_of_grid_points() / 2] = 1.0;
+
+    get(get<Inactive<grmhd::ValenciaDivClean::Tags::TildeD>>(
+        subcell_vars))[subcell_mesh.number_of_grid_points() / 2] = -1.0e-20;
+    CHECK(grmhd::GhValenciaDivClean::subcell::DgInitialDataTci::apply(
+        dg_vars, subcell_vars, 1.0e100, epsilon, exponent, dg_mesh,
```

Comment:
persson exponent was not changed this line?

Reply:
It doesn't need to be. I checked which ones needed to be changed and only changed the necessary ones

Reply:
Yep, I got it.